### PR TITLE
Split gizmos.py into submodules, final task of #218

### LINKED ISSUE #218: Code Cleanup
As the addon has grown quite a bit and new contributors are jumping onboard there's some cleanup overdue. This task is used to keep track of code cleanup efforts.

Feel free to suggest additional topics/areas in the comments.

## Structure
- [x] #219
- [x] Improve registration flow #223
- [x] Slip out utilities and constants https://github.com/hlorus/CAD_Sketcher/commit/b73d41b1ce5749abb988533e4bb6f215e468353e#diff-8656a273c89d5812379185ade358273fa3d93e2cb8df832c66cec928eef9809a
- [x] Split out blender classes into into individual files (PropertyGroups, Operators) #229
- [x] Split out UI definitions into individual files (ui.py, workspacetools.py)
- [x] Move all keymap definitons to keymaps.py
- [x] Split out gizmos.py into individual files

## Style
- [x] #248 

(docstrings and type hints tracked in #335 and #171 respectively)

### PR DESCRIPTION
**Pull request by Cheaterman**
Closes #218 when merged.

### CODE REVIEW

File: gizmos/__init__.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            continue
+        list = getattr(constrs, name)
+
+        for entity in list:
+            yield entity
+
+
+# TODO: This could already Skip entities and constraints that are not active
+# TODO: only store indices instead of actual objects
+def constraints_mapping(context):
```
Comment (hlorus) [inline gizmos/__init__.py:34]: Move to gizmo/utilities.py

**Review by hlorus — CHANGES_REQUESTED** [review]
Looks generally good, i've commented a few details inline.

File: gizmos/__init__.py (hunk 1 above)
Comment (Cheaterman) [reply gizmos/__init__.py:34]: Or delete it, apparently it's dead code.

**Review by hlorus — APPROVED** [review]
Perfect, looks good now!

**Pull request merged**